rero_ils/modules/utils.py

```diff
@@ -50,6 +50,17 @@ def wrapper(*args, **kwargs):
     return caching
 
 
+def memoized(timeout=50):
```

Comment:
Please correct the doc string. Can you add more details on it and mention this in the commit message. Thanks.

Reply:
ok